formhero/apps/admin.py

```diff
@@ -1,3 +1 @@
-from django.contrib import admin
-
 # Register your models here.
```

Comment:
@szymonzmyslony We could be cleaner and simply delete this file, rather than leaving it (essentially) empty.